app/(candidate)/dashboard/voter-records/[type]/components/ScheduleFlow.js

```diff
@@ -9,8 +9,10 @@ import ScheduleAddScriptFlow from 'app/(candidate)/dashboard/voter-records/[type
 import ScheduleFlowScheduleStep from './ScheduleFlowScheduleStep';
 import ScheduleFlowComplete from './ScheduleFlowComplete';
 import ScheduleFlowImageStep from './ScheduleFlowImageStep';
+import CloseConfirmModal from './CloseConfirmModal';
 import { buildTrackingAttrs } from 'helpers/fullStoryHelper';
 import { scheduleVoterMessagingCampaign } from 'helpers/scheduleVoterMessagingCampaign';
+import { isEqual } from 'lodash';
```

Comment:
do we have lodash in package.json? I don't see it
I don't like using it because it usually imports too much bloat. Can you just create a help instead of using lodash?
specifically, you should do here `import isEqual from 'lodash/isEqual'`

but I rather just not use lodash at all if possible